src/Sparse/Row.jl

```diff
@@ -535,13 +562,18 @@ function div(A::SRow{T}, b::Integer) where T
   return div(A, base_ring(A)(b))
 end
 
+@doc raw"""
+  divexact(A::SRow, b::NCRingElem; check::Bool = true) -> SRow
+
+Returns the left divexact $A/b$
+"""
 function divexact(A::SRow{T}, b::T; check::Bool=true) where T
```

Comment:
Hmm, IMHO this code is missing a `divexact_left` method. I suggest to rename this one (also in the docstring), and perhaps restrict `T` to `NCRingElem`.
```suggestion
function divexact_left(A::SRow{T}, b::T; check::Bool=true) where T <: NCRingElem
```

And then implement `divexact` by delegating to `divexact_left`... possibly with `T` restricted to `RingElem` -- but I'd also be fine with having it general, but then the docstring for this `divexact` method should explicitly say that it delegates to `divexact_left`.
```
divexact(A::SRow{T}, b::T; check::Bool=true) where T <: RingElem = divexact_left(A, b; check)
```

Of course the delegation could also be done in the other direction, that's how it is done right now in AA for "ring elements", but I think the situation there is a bit different: there `divexact` is already implemented for commutative rings and we just allow `divexact_left` to work; but for non-commutative rings / ring elements, `divexact` just is never defined... So I don't think this pattern matters here, hence my suggestion above.
```
divexact_left(x::T, y::T; check::Bool=true) where T <: RingElement = divexact(x, y; check=check)

divexact_right(x::T, y::T; check::Bool=true) where T <: RingElement = divexact(x, y; check=check)
```